Doc/template#10 - Add scenario-management template to doc and relnotes (#529)
Avaiga/taipy-doc: docs/manuals/cli/create.md

- The first question defines the application root folder as "new_application"
- In the second and third questions, we set the main Python file of the application as "app.py" and the web page's title
  as "App Title".

> - In the 4th question, we clarify that we want a Taipy multi-page GUI application with 3 pages, and the page names are

Reviewer: ```suggestion
- In the 4th question, we clarify that we want a Taipy multi-page GUI application with three pages, and the page names are
```

  "slide_1", "slide_2", and "slide_3". Please note that if the names are not clarified, the default name will be
  "page_1", "page_2", and so on.
- In the 5th question, we clarify that we want to use scenario management, so the application should include the
  Taipy Core service.

> - In the 6th question, we choose the default answer is No, which means that we don't want to use Rest API, so the

Reviewer: ```suggestion
- In the 6th question, we chose the default answer is No, meaning we don't want to use Rest API, so the
```

  application should not include the Taipy Rest API service.

Finally, we changed the directory (`cd`) to our newly created folder and started the application by running
`python app.py`.

## From a specific template

You can specify another template using the `--template` option.

```console
$ taipy create --template scenario-management
Application root folder name [taipy_application]: new_application
Application main Python file [main.py]: app.py
Application title [Default title]: App Title
Does the application use TOML Config? [No]:
$ cd ./new_application
$ python app.py
```

In this example, we scaffold a new Taipy application using the "scenario-management" template, which utilizes

> [a scenrio selector](./../gui/corelements/scenario_selector.md) to allow creating, managing, and running scenarios

Reviewer: ```suggestion
[a scenario selector](./../gui/corelements/scenario_selector.md) to allow creating, managing, and running scenarios
```
